src/please.go

```diff
@@ -643,14 +642,10 @@ var buildFunctions = map[string]func() bool{
 		return true
 	},
 	"changes": func() bool {
-		// Temporarily set this flag on to avoid fatal errors from the first parse.
```

Comment:
Now I see this, I'm slightly worried about losing this. I think the condition on line 98 of shell_output.go will now pass, which would cause the build to fail early here which isn't what we want.

I guess ideally we should refactor out that `os.Exit` call, it's obviously not very friendly...

Reply:
OK I'm going to merge this and then try to factor that into something nicer than calling `os.Exit`.